### DISCUSSION TITLE
Refund

### DISCUSSION THREAD

**Discussion by guanjiajia123** [community/community#121391] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I accidentally purchased a year's worth of charges, and PayPal deducted $100 directly. I wanted to get a refund, but I couldn't find a way to do so. I submitted a ticket and there was no reply. Please ask the staff to help me handle it as soon as possible.

**Comment by davecheney** [comment, marked as answer]
I am sorry, we cannot assist you with this issue via this forum. Please contact support https://support.github.com/contact

**Reply by nataliepan** [reply]
That's where I went there was only options to ask copilot or create a ticket
